webpack.config.js

```diff
@@ -74,6 +74,27 @@ const babelLoader = {
 	},
 };
 
+const preSassLoaders = [
+	MiniCssExtractPlugin.loader,
+	'css-loader',
+	{
+		loader: 'postcss-loader',
+		options: {
+			plugins: _.compact( [ require( 'autoprefixer' ), ! isDevelopment && require( 'cssnano' ) ] ),
+		},
+	},
+];
+
+const sassLoader = {
+	loader: 'sass-loader',
+	options: {
+		includePaths: [ path.join( __dirname, 'client' ) ],
+	},
+};
+
+// When styles-namespacing is enabled, these are the files we want to namespace
+const styleNamespaceDirectories = [ path.join( __dirname, 'client', 'components' ) ];
```

Comment:
From the outside perspective, it feels a little inconsistent to namespace only components, and not namespace the block stylesheets. If I'm an external plugin/theme developer, I'd be confused why some stylesheets have the prefix and others don't. What do you think?